index.js

```diff
@@ -40,26 +40,27 @@ function initialize(options) {
     sdkKey,
     datafile,
     logLevel,
-    datafileOptions,
   } = options;
 
 
   const defaultLogger = require('@optimizely/optimizely-sdk').logging;
-  const manager = new DatafileManager({
-    sdkKey,
+
+  const optimizelyClient = OptimizelySdk.createInstance({
+    logger: defaultLogger.createLogger({
+      logLevel: logLevel
+    }),
     ...options,
-    ...datafileOptions,
   });
 
   function updateDatafile() {
     datafile = manager.get()
     datafile._lastUpdated = new Date();
-    console.log('[Optimizely] Datafile Updated!');
+    console.log('[OPTIMIZELY] Datafile Updated!');
```

Comment:
Just to match the rest of the SDK.

Reply:
Thoughts on making this `optimizelyClient.logger.info`? We could also remove this because the SDK does log already when the datafile is successfully updated